proto/store/idp.proto

```diff
@@ -63,3 +63,17 @@ message FieldMapping {
   // Email is the field name of primary email in 3rd-party idp user info. Required.
   string email = 3;
 }
+
+message IdentityProviderUserInfo {
+  // Identifier is the value of the unique identifier in 3rd-party idp user info.
+  string identifier = 1;
+
+  // DisplayName is the value of display name in 3rd-party idp user info.
+  string display_name = 2;
+
+  // Email is the value of primary email in 3rd-party idp user info.
+  string email = 3;
+
+  // Raw contains original fields returned by the Identity Provider.
```

Comment:
Instead of storing raw, we should transform the raw data into semi-structured data for being useful information.

Reply:
We cannot represent all user information through a single structure. For example, Google/GitHub's user info API has a different response structure. 

And we only use `identifier`, `display_name` and `email` from raw data at this stage. There already has this structure.

Reply:
We can use different fields in the message?